### DISCUSSION TITLE
tmpfs no increased RAM usage

### DISCUSSION THREAD

**Discussion by jhallenb** [blakeblackshear/frigate#1498] [Ask A Question] [answered]
I am running frigate in proxmox (on a lxc container that runs docker).
I used docker-compose and the latest 1.29.2 version of docker-compose.
The docker-compose file is copied from the frigate howto.

It runs fine, but I expected the RAM usage to go up when specifying 1 gig as tmpfs.
It does not according to the proxmox RAM graph.

Is this expected?
How can I verify that frigate uses the tmpfs.

Thanks for the great work
/Johan

**Comment by blakeblackshear** [comment, marked as answer]
I wouldn't expect to see the usage jump until its actually being used. Check `/api/stats` and see what it says for `/dev/shm` under storage.

**Reply by jhallenb** [reply]
Thanks
Seems 80Mb was in use.
Thanks for the fast reply.

**Reply by flaektrem** [reply]
> I wouldn't expect to see the usage jump until its actually being used. Check `/api/stats` and see what it says for `/dev/shm` under storage.

@blakeblackshear How can I verify that this ```/dev/shm``` is written to RAM and not disk? I'm seeing high "Disc I/O in Proxmox and I'm afraid that my disc is beeing used instead of RAM

![Screenshot 2022-01-03 at 13-39-48 proxmox - Proxmox Virtual Environment](https://user-images.githubusercontent.com/58172129/147931663-aad0d521-e793-4e2e-9dac-9ca5b8997bd4.png)
![Screenshot 2022-01-03 at 13-44-34 proxmox - Proxmox Virtual Environment](https://user-images.githubusercontent.com/58172129/147932024-48157d44-594b-4b09-ba6a-7d425a69cacc.png)
![Screenshot 2022-01-03 at 13-44-21 proxmox - Proxmox Virtual Environment](https://user-images.githubusercontent.com/58172129/147932030-65d9de71-bcce-4978-95b2-c9f626bd9c14.png)
.
